specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2851 @@
+            "description": "A successful call result",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTaskResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Entity Linking Request": {
+            "$ref": "./examples/SuccessfulEntityLinkingRequest.json"
+          },
+          "Successful Entity Recognition Request": {
+            "$ref": "./examples/SuccessfulEntityRecognitionRequest.json"
+          },
+          "Successful Key Phrase Extraction Request": {
+            "$ref": "./examples/SuccessfulKeyPhraseExtractionRequest.json"
+          },
+          "Successful PII Entity Recognition Request": {
+            "$ref": "./examples/SuccessfulPiiEntityRecognitionRequest.json"
+          },
+          "Successful Language Detection Request": {
+            "$ref": "./examples/SuccessfulLanguageDetectionRequest.json"
+          },
+          "Successful Sentiment Analysis Request": {
+            "$ref": "./examples/SuccessfulSentimentAnalysisRequest.json"
+          }
+        }
+      }
+    },
+    "/analyze-text/jobs": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "description": "Submit a collection of text documents for analysis. Specify one or more unique tasks to be executed as a long-running operation.",
+        "operationId": "AnalyzeText_SubmitJob",
+        "summary": "Submit text analysis job",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "Collection of documents to analyze and one or more tasks to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextJobsInput"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "A successful call results with an Operation-Location header used to check the status of the analysis job.",
+            "headers": {
+              "Operation-Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Submit Analysis Job Request": {
+            "$ref": "./examples/SuccessfulAnalyzeTextJobsMultipleTaskSubmitRequest.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/analyze-text/jobs/{jobId}": {
+      "get": {
```

Comment:
Is `delete` planned to be added in this preview or the next?

Reply:
It has been moved to a TBD preview.  The original plan was to include it here, then it got bumped to March, but now we may actually be skipping the March preview for TA to focus more resources on the April preview.

I will know more about the planning changes on Thursday morning, so I can share more details with you offline once I know for sure.  It will definitely not be included in this preview, though.